src/components/DashboardPage/ScanDashboardData.js

```diff
@@ -2,21 +2,23 @@ import React, { Component } from 'react';
 import { SCAN_ACTIONS } from '../../redux/actions/scanActions';
 
 const mapStateToProps = reduxState => ({
-    scanInfo:reduxState.scanInfo
+    scanReducer:reduxState.scanReducer
 });
-
 class ScanDashboardData extends Component {
+    componentDidMount() {
+        this.props.dispatch({ type: SCAN_ACTIONS.FETCH_SCAN});
+    }
     render() {
     
     return (
       <div>
-          Sites Scanned : {this.props.scanInfo.length}
+          Sites Scanned : {this.props.scanReducer.length}
```

Comment:
Check out the site router for the documentation on what this object is going to look like. Also, same comment as profileReducer.